acceptor_test.go

```diff
@@ -25,7 +26,7 @@ func TestAccept(t *testing.T) {
 		require.NotNil(t, conn)
 
 		acceptor := xconn.WebSocketAcceptor{}
-		session, err := acceptor.Accept(conn)
+		session, err := acceptor.Accept(conn, time.Duration(0), time.Duration(0))
```

Comment:
this would then just become
```suggestion
		session, err := acceptor.Accept(conn, nil)
```